examples/working-directly-with-json/working_directly_with_json.md

```diff
@@ -1,11 +1,10 @@
-# Working directly with JSON
+# Work directly with JSON
```

Comment:
We have multiple BBEs that have similar titles. E.g., [Ignoring return values and errors](https://ballerina.io/learn/by-example/ignoring-return-values-and-errors.html), [Controlling openness](https://ballerina.io/learn/by-example/controlling-openness.html), etc. Can we leave these as is and do the changes in a separate PR if required?

Reply:
Fixed.